overrides.css

```diff
@@ -1,149 +1,9 @@
 /* Deno Doc Overrides */
 
-div.ddoc {
-  @apply flex h-screen p-4 pt-2 lg:p-2 lg:pt-0;
-}
-
-/* First column: fixed width of 290px */
-.ddoc > :first-child {
-  @apply w-[290px] flex-shrink-0;
-}
-
-/* Second column: takes up the remaining space */
-.ddoc > :nth-child(2) {
-  @apply flex-grow;
-}
-
-.toc {
-  @apply hidden;
-}
-
-.ddoc > .toc {
-  @apply hidden md:flex md:flex-col !important;
-}
-
-.ddoc > div > div#content > div.toc {
-  @apply hidden xl:flex xl:flex-col !important;
-}
-
-.ddoc > .toc {
-  @apply h-full max-h-screen border-r border-gray-000;
-}
-
-.ddoc .documentNavigation > h3 {
-  margin: 0.75rem 0.75rem 0.5rem !important;
-  padding-bottom: 0.1rem;
-  @apply text-gray-4 font-normal text-xs border-b border-gray-000 leading-normal uppercase !important;
-}
-
-.ddoc .documentNavigation > ul {
-  @apply flex-grow overflow-y-auto;
-}
-
-.ddoc .documentNavigation > ul > li {
-  @apply mt-0.5 mx-3;
-}
-
-.ddoc .documentNavigation > ul > li > a {
-  @apply text-gray-3 text-sm py-1.5 px-3.5 hover:bg-blue-50 rounded-lg hover:no-underline !important;
-}
-
-.ddoc > div:not(.toc) {
-  @apply flex flex-col;
-}
-
-#content {
-  @apply mt-4 flex flex-row justify-between !important;
-}
-
-#content > main {
-  @apply pb-0 flex flex-col gap-3 flex-grow !important;
-}
-
-#content > .toc {
-  @apply flex-shrink-0 min-w-[250px] max-w-[300px] !important;
-}
-
 #content > main > section + div {
   @apply max-w-[75ch] !important;
 }
 
-.ddoc .markdown {
-  @apply max-w-[40ch] sm:max-w-screen-sm md:max-w-screen-md lg:max-w-[75ch] !important;
-}
-
-/* This comes from gfm css, should be replaced with something in our color palette once we establish it. */
-.ddoc .markdown :not(pre) > code {
-  background-color: var(--color-neutral-muted) !important;
-}
-
-.ddoc .markdown pre {
-  @apply border-gray-000 !important;
-}
-
-.ddoc .max-w-prose {
-  @apply max-w-[75ch] !important;
-}
-
-.ddoc > div > div#content > div.toc > div > nav.documentNavigation > h3 {
-  @apply hidden !important;
-}
-
-.ddoc > div > div#content > div.toc > div > nav.documentNavigation > ul {
-  @apply border-l border-gray-000 text-gray-1 text-[.8rem] leading-none !important;
-}
-
-.ddoc
-  > div
-  > div#content
-  > div.toc
-  > div
-  > nav.documentNavigation
-  > ul
-  > li:has(> a) {
-  @apply pb-0 !important;
-}
-
-.ddoc nav.documentNavigation li:has(> ul) {
-  @apply mt-0 !important;
-  /* CSS styles here */
-}
-
-.ddoc
-  > div
-  > div#content
-  > div.toc
-  > div
-  > nav.documentNavigation
-  > ul
-  > li
-  > ul
-  > li {
-  @apply mt-1 !important;
```

Comment:
Some of these long document navigation styles should move or else we lose the formatting of the document navigation

<img width="274" alt="Screenshot 2024-07-11 at 9 51 30 AM" src="https://github.com/denoland/deno-docs/assets/776987/7097aadc-713c-4739-a627-b6f16ff1cb8a">
<img width="273" alt="Screenshot 2024-07-11 at 9 49 19 AM" src="https://github.com/denoland/deno-docs/assets/776987/60f63931-e41b-42e2-a80b-66e35f6256cd">